tmk_core/avr.mk

```
@@ -246,6 +246,19 @@ endef
 usbasp: $(BUILD_DIR)/$(TARGET).hex check-size cpfirmware
 	$(call EXEC_USBASP)
 
+BOOTLOADHID_PROGRAMMER ?= bootloadHID
+
+define EXEC_BOOTLOADHID
+	# bootloadHid executable has no cross platform detect methods
+	# so keep running bootloadHid if the output contains "The specified device was not found"
+	until $(BOOTLOADHID_PROGRAMMER) -r $(BUILD_DIR)/$(TARGET).hex 2>&1 | tee /dev/stderr | grep -v "device was not found"; do\
+		echo "Trying again in 5s." ;\
```

Would we update this to be more consistent:
```suggestion
		echo "Error: Bootloader not found. Trying again in 5s." ;\
```